edx_event_bus_kafka/internal/consumer.py

```diff
@@ -513,8 +519,8 @@ def record_event_consuming_error(self, run_context, error, maybe_message):
         try:
             # This is gross, but our record_exception wrapper doesn't take args at the moment,
             # and will only read the exception from stack context.
-            raise Exception(error)
-        except BaseException:
+            raise EventConsumptionException(error)
+        except EventConsumptionException:
             self._add_message_monitoring(run_context=run_context, message=maybe_kafka_message, error=error)
             record_exception()
```

Comment:
So will this end up recording the `EventConsumptionException`, or will it record the type of `error`?

Reply:
(Well, I guess if we lose the type of `error`, that's _already_ happening, so no complaint here.)

Reply:
Yeah, we were losing the type of error previously. This allows us to at least see a `EventConsumptionException` recorded as the `error.class` in New Relic.